tensorflow/contrib/learn/python/learn/estimators/estimator.py

```diff
@@ -403,8 +403,6 @@ class BaseEstimator(sklearn.BaseEstimator, evaluable.Evaluable,
   Users should not instantiate or subclass this class. Instead, use an
   `Estimator`.
   """
-  __metaclass__ = abc.ABCMeta
```

Comment:
I did not decorate this class with `@six.add_metaclass`, because its base classes already have `abc.ABCMeta` as the metaclass.